### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by loriab** [PCMSolver/pcmsolver#39]
I don't think PCMSolver is Python3 compatible. Please to remedy this, as needed for Psi4 1.1. Here's the first error I get.

```
Traceback (most recent call last):
  File "stage//hrw-temp/install-psi4/bin/psi4", line 204, in <module>
    content = psi4.process_input(content)
  File "/hrw-temp/objdir27/stage/hrw-temp/install-psi4/lib/psi4/driver/inputparser.py", line 726, in process_input
    temp = re.sub(pcm, process_pcm_command, temp)
  File "//miniconda/envs/texternals35/lib/python3.5/re.py", line 182, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/hrw-temp/objdir27/stage/hrw-temp/install-psi4/lib/psi4/driver/inputparser.py", line 395, in process_pcm_command
    import pcmsolver
  File "/hrw-temp/objdir27/stage//hrw-temp/install-psi4/lib//pcmsolver/pcmsolver.py", line 63, in <module>
    import getkw
  File "/hrw-temp/objdir27/stage//hrw-temp/install-psi4/lib//pcmsolver/getkw.py", line 70
    raise TypeError, 'Not a Section or Keyword'
                   ^
SyntaxError: invalid syntax
```

**Comment by robertodr** [comment]
No, it definitely isn't Python 3 compatible. I am working on it on your fork, it's going to be easier than doing it on the mess I currently have on my development branch :disappointed:

**Comment by robertodr** [comment]
Closed by #38
